zookeeper-server/src/test/java/org/apache/zookeeper/audit/Slf4JAuditLoggerTest.java

```diff
@@ -322,8 +312,7 @@ private static void verifyLog(String expectedLog, String log) {
         String searchString = " - ";
         int logStartIndex = log.indexOf(searchString);
         String auditLog = log.substring(logStartIndex + searchString.length());
-        assertEquals(expectedLog, auditLog);
-
+        Assert.assertTrue(auditLog.endsWith(expectedLog));
```

Comment:
Just curious, why with log4j it's "equals", and with logback it's "endWith"? Does logback add something in the front?

Reply:
yeah logback logging format is little different hence we had to check with endsWith. I verified logback had all existing things from log4j in logs but just few extra things like classname in detail or few formatting changes.